### ISSUE TITLE
Adding a trackable cuda feature to cudatoolkit

### ISSUE THREAD

**Issue by carterbox** [ContinuumIO/anaconda-issues#11272]
This is a feature request for the cudatoolkit package.

I'd like cudatoolkit to export a `cuda` [feature](https://docs.conda.io/projects/conda-build/en/latest/concepts/features.html#concepts-features). If this existed, I could track the cuda feature in my build recipes and automatically decide whether to install a cuda or no-gpu build to an existing environment.

**Comment by msarahan** [comment]
no, features are not a great idea.  The solver logic ends up doing really undesirable things.  You can read more in some recently added docs:

PR: https://github.com/conda/conda/pull/9216
Docs: https://docs.conda.io/projects/conda/en/latest/user-guide/concepts/packages.html?highlight=mutex#mutex-metapackages

You want mutual exclusivity of some implementation detail, but my experience has been that you do not want the aggressive group behavior that features have.  We will likely deprecate features completely soon.  track_features will stay, for the priority reasons mentioned in those docs.
